webapp/app/sensors/routes.py

```diff
@@ -50,6 +50,7 @@ def sensor_list():
     )
     sensors = db.session.execute(query).fetchall()
     sensors_arr = query_result_to_array(sensors, date_iso=False)
+    print(f"NUMBER OF SENSORS IS {len(sensors_arr)}")
```

Comment:
Is this line meant to stay?

Reply:
Similarly you're right, this was just for debugging - removed now :)